<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>signalamplifier</class>
 <widget class="QWidget" name="signalamplifier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>900</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>420</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <item>
    <property name="text">
     <string>根</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>左结点</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>右结点</string>
    </property>
   </item>
  </widget>
  <widget class="QLineEdit" name="parentO">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="weightO">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>530</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="selfO">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>200</y>
     <width>120</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>13</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>161</width>
     <height>50</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前结点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>310</y>
     <width>161</width>
     <height>40</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>父结点</string>
   </property>
  </widget>
  <widget class="QLabel" name="label3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结点类型</string>
   </property>
  </widget>
  <widget class="QLabel" name="label4">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>530</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>宋体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>结点权值</string>
   </property>
  </widget>
  <widget class="QLabel" name="label5">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>660</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>13</pointsize>
    </font>
   </property>
   <property name="text">
    <string>当前节点个数为：</string>
   </property>
  </widget>
  <widget class="QLabel" name="countShow">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>650</y>
     <width>45</width>
     <height>45</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微软雅黑</family>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
